### DISCUSSION TITLE
"Open polygon" terminology

### DISCUSSION THREAD

**Discussion by stla** [CGAL/cgal#7247] [General]
In [this chapter](https://doc.cgal.org/latest/Boolean_set_operations_2/index.html), in the reference manual, the terminology "open polygon" is used multiple times. However it is not defined, at least not defined in the user manual. It seems to be a synonym of "polygon without hole".

**Comment by efifogel** [comment]
Open polygon refers to the polygon without its boundary or the interior.
I guess we can add a sentence with the explicit definition.
   ____  _        ____             _
  /_____/_) o    /__________  __  //
 (____ (   (    (    (_/ (_/-(-'_(/
                         _/

**Reply by stla** [reply]
Ah ok, "open" in the topological sense. Thanks.
